packages/components/src/VirtualizedList/CellMappedData/CellMappedData.component.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import isArray from 'lodash/isArray';
+
+import { defaultColumnConfiguration } from '../Content.component';
+
+const { cellRenderer: DefaultCellRenderer } = defaultColumnConfiguration;
+
+function CellMappedData(props) {
+	const { cellData, columnData } = props;
+	const { valuesMap, ...restColumnData } = columnData;
+
+	let cellContent;
+
+	if (isArray(cellData)) {
+		cellContent = cellData
+			.map(value => valuesMap[value] || value || null)
+			.filter(value => !!value)
+			.sort((a, b) => a.localeCompare(b))
```

Comment:
I assume this new cell doesn't support number.
If so it's ok, just to make sure that you don't need/want to support numbers.